modules/oe_theme_content_organisation/src/Plugin/PageHeaderMetadata/OrganisationContentType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_organisation\Plugin\PageHeaderMetadata;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_theme_helper\Plugin\PageHeaderMetadata\NodeViewRoutesBase;
+
+/**
+ * Page header metadata for the OpenEuropa Organisation content entity.
+ *
+ * @PageHeaderMetadata(
+ *   id = "organisation_content_type",
+ *   label = @Translation("Metadata extractor for the OE Organisation Content content type"),
+ *   weight = -1
+ * )
+ */
+class OrganisationContentType extends NodeViewRoutesBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(): bool {
+    $node = $this->getNode();
+
+    return $node && $node->bundle() === 'oe_organisation';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMetadata(): array {
+    $metadata = parent::getMetadata();
+    $node = $this->getNode();
+
+    if (!($node->get('oe_summary')->isEmpty())) {
```

Comment:
From my point of view we should implement a method like setIntroductionMetaData($field_name = 'oe_summary') in the NodeViewRoutesBase because this code is re-used in all content types.

Reply:
If NodeViewRoutesBase has been updated then this method should be adjusted accordingly.

Reply:
done